integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/webhooks/subscription/SubscriptionRepository.java

```diff
@@ -0,0 +1,28 @@
+package no.difi.meldingsutveksling.webhooks.subscription;
+
+import no.difi.meldingsutveksling.domain.webhooks.QSubscription;
+import no.difi.meldingsutveksling.domain.webhooks.Subscription;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.querydsl.QuerydslPredicateExecutor;
+import org.springframework.data.querydsl.binding.QuerydslBinderCustomizer;
+import org.springframework.data.querydsl.binding.QuerydslBindings;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.data.repository.query.Param;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.springframework.data.repository.query.Param'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AUselessImportCheck)